packages/runner-ct/src/app/SpecList/SpecList.tsx

```diff
@@ -194,6 +195,10 @@ export const FileTree: React.FC<FileTreeProps> = (props) => {
     return render(match.indexes)
   }
 
+  if (props.specs.length < 1) {
+    return <NoSpec message="No spec found" />
```

Comment:
super nit: Maybe centralize the strings in `NoSpec`, and choose one through props.